bin/fontbakery-family-html-snippet.py

```diff
@@ -0,0 +1,160 @@
+"""
+Generate a test html snippet for a family hosted on fonts.google.com
+
+This script works well for quickly producing test cases using jsbin.
+
+$ fontbakery family-html-snippet "Exo" "Hello World"
+>>> ...
+<html>
+  <head>
+    <link href="https://fonts.googleapis.com/css?family=Exo"/rel="stylesheet">
+    <style>
+      .g{font-family: 'Exo'; font-weight:400;}
+      .h{font-family: 'Exo'; font-weight:400; font-style: italic;}
+      .m{font-family: 'Exo'; font-weight:700;}
+      .n{font-family: 'Exo'; font-weight:700; font-style: italic;}
+    </style>
+  </head>
+  <body>
+    <p class='g'>Hello World</p>
+    <p class='h'>Hello World</p>
+    <p class='m'>Hello World</p>
+    <p class='n'>Hello World</p>
+  </body>
+</html>
+"""
+import json
+import requests
+from argparse import ArgumentParser
+
+GF_API = 'http://tinyurl.com/m8o9k39'
```

Comment:
explicitely using https://www.googleapis.com/webfonts/v1/webfonts?key=AIzaSyDK4Jz71F7DQCrUhXYaF3xgEXoQGLDk5iE here would be better !